sale_to_purchase_order/models/procurement_batch_generator.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Authors: Laetitia Gangloff
+#    Copyright (c) 2015 Acsone SA/NV (http://www.acsone.eu)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+from openerp import api, fields, models
+
+
+class ProcurementBatchGenerator(models.TransientModel):
+    _inherit = 'procurement.batch.generator'
+
+    @api.model
+    def _is_product_to_add(self, order_line):
+        """ You can add rules to determine if the product of the order line
+        should be in the default line
+        """
+        return True
```

Comment:
Can you extend the functional use case where this hook is necessary?

An example should be cool!